wagtail/admin/panels/page_utils.py

```diff
@@ -23,7 +23,10 @@ def set_default_page_edit_handlers(cls):
                 attrs={
                     "placeholder": format_lazy(
                         "{title}*", title=gettext_lazy("Page title")
-                    )
+                    ),
+                    "data-controller": "w-sync",
+                    "data-w-sync-target-value": "body:not(.page-is-live) #id_slug",
+                    "data-action": "focus->w-sync#check blur->w-sync#apply change->w-sync#clear",
```

Comment:
```suggestion
                    "data-action": "focus->w-sync#check blur->w-sync#apply",
```

I do not think we need the clear behaviour here, this could cause some confusing issues.